carrow.c

```diff
@@ -44,6 +47,12 @@ struct evbag {
 };
 
 
+struct asapbag {
```

Comment:
Use `struct evbag` instead. We need to schedule coroutines to run as soon as possible.